dapps/vue-dapp-auth/assets/scss/themes/_dark.scss

```diff
@@ -0,0 +1,38 @@
+@mixin dark-mode {
```

Comment:
Were the dark/light theme mixins sourced from somewhere that we can reference back to or are these yours?

Reply:
No, these are mine

Reply:
Thanks for clarifying 👍